integration-cli/docker_cli_run_test.go

```diff
@@ -1128,6 +1128,13 @@ func (s *DockerSuite) TestRunWithBlkioInvalidWeight(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithBlkioInvalidWeightDevice(c *check.C) {
```

Comment:
Hi @jfrazelle I planted to add a valid one. But it seems it's not easy to find out what block device does the janky have.